# Goal
geometry_msgs/PoseStamped target_pose
---
# Result. Empty in navigation
---
# Feedback. Current robot position
geometry_msgs/PoseStamped base_position
string text
